### PR TITLE
Fix/dashed box component

### LINKED ISSUE #1152: [BUG] Dashed box component does not respect line breaks
**Describe the bug**
Dashed box component does not respect line breaks in text. 

**App version**
v0.13.0

**Debug Template**
[debug_dashed_box_paragraphs](https://docs.google.com/spreadsheets/d/1rtqqvxb50wGa00AJcF3GcTeUdxLky2pcK84mA8G0Few/edit#gid=383748944)
image

**Screenshots and videos**
image

### PR DESCRIPTION
**Pull request by kwAsant**
PR Checklist

- [x] PR title descriptive (can be used in release notes)

## Description

Addresses errors where line breaks were not considered within the dashed-box component.

## Git Issues

Closes #1152 

## Screenshots/Videos

[debug_dashed_box_paragraphs](https://docs.google.com/spreadsheets/d/1rtqqvxb50wGa00AJcF3GcTeUdxLky2pcK84mA8G0Few/edit#gid=383748944)

image

### CODE REVIEW

**Review by esmeetewinkel — APPROVED** [review]
Functional test passed.

File: src/app/shared/components/template/components/dashed-box/dashed-box.component.scss [hunk 1]
```diff
@@ -11,11 +11,16 @@
...
-    .text {
+    .text{
+      white-space: pre-line; 
       text-align: center;
       font-size: var(--font-size-text-medium);
       line-height: var(--line-height-text-largest);
-    }
+
+      p {
+        margin-bottom: 0.75em 0 ;
```
Comment (jfmcquade) [inline src/app/shared/components/template/components/dashed-box/dashed-box.component.scss:21]: I don't think this CSS rule is doing anything so it's not needed. As far as I can tell, the text that gets rendered is passed in as `innerHTML`, so there are no `<p>` elements inside `.text`. I would recommend deleting this rule unless there's a reason for it

**Review by jfmcquade — CHANGES_REQUESTED** [review]
Good work, this solves the issue. I've added a comment inline where I think you can remove some code.

**Review by jfmcquade — APPROVED** [review]
LGTM

**Pull request merged**